test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -443,6 +443,20 @@ static int test_overall_capabilities(void)
 				CU_ASSERT(per_level->max_priority != 0);
 				return -1;
 			}
+
```

Comment:
Validation test should check capabilities before configuring rates and burst
sizes in traffic_mngr_test_shaper_profile() and in set_shaper().

Also, the traffic manager example app should check the new capabilities before configuring things.